uber/templates/preregistration/attendee_donation_form.html

```diff
@@ -5,7 +5,7 @@
 <div class="masthead">
 </div>
 <div class="panel panel-default">
-{% if attendee.paid == c.NOT_PAID or attendee.overridden_price %}
```

Comment:
As someone who can't recall exactly much of anything, seeing what I paid for my badge would in fact make me happy.